packages/union-component/src/__tests__/DonationWidget.spec.tsx

```diff
@@ -78,6 +101,12 @@ test('send a donation request with all provided information', async () => {
   expect(submitBtn).toBeDisabled();
 
   expect(screen.getByText(widgetTitle)).toBeInTheDocument();
+  await waitFor(() =>
```

Comment:
After this you should have something like:

```
userEvent.selectOptions(
    screen.getByRole('combobox', { name: /specific fund/i }),
    'anti-eviction' // value of an option
)
```

https://github.com/testing-library/user-event#selectoptionselement-values

**Please do that before merging**